### DISCUSSION TITLE
[SC]: chiarimento data validità

### DISCUSSION THREAD

**Discussion by Hdemografici** [italia/ansc#988] [Supporto servizi cooperativi] [answered]
### Ambiente

pre-produzione

### Servizio

R014 - Servizi cooperativi per la gestione data di validità verso anagrafe

### idOperazione

n/a

### Provenienza segnalazione

Software House

### Comune di riferimento e operatore

assente

### Descrivi il problema

Buongiorno, se ho capito bene il servizio R014 serve per indicare la data di validità dell'atto per quanto riguarda il collegamento con ANPR e quindi la generazione delle notifiche, quindi escluso il caso della cittadinanza con decorrenza il giorno successivo a quello di redazione atto, si può indicare la data del giorno come data di validità, è corretto?

**Comment by gbuemi** [comment, marked as answer]
Buongiorno @Hdemografici.
la data validità può certamente essere quella odierna o successiva ma la decide manualmente l'USC, in base all'evento che sta registrando.

Un saluto,
Supporto ANSC
